pkg/migrations/sql/idempotent/base.sql

```diff
@@ -1456,6 +1455,8 @@ BEGIN
         FROM SCHEMA_CATALOG.ids_epoch LIMIT 1
         INTO last_epoch_time, deletion_epoch, next_epoch;
 
+    COMMIT;
```

Comment:
Please add a comment
```suggestion
    -- We commit here to avoid a lock escalation on the ids_epoch table when we later take an access exclusive lock 
    -- on it
    COMMIT;
```